presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -578,37 +577,19 @@ public static PlanTestSymbol anySymbol()
         return new AnySymbol();
     }
 
-    public static ExpectedValueProvider<FunctionCall> functionCall(String name, List<String> args)
-    {
-        return new FunctionCallProvider(QualifiedName.of(name), toSymbolAliases(args));
-    }
-
-    public static ExpectedValueProvider<FunctionCall> windowFunctionCall(String name, List<String> args)
-    {
-        return new FunctionCallProvider(true, QualifiedName.of(name), false, toSymbolAliases(args));
-    }
-
-    public static ExpectedValueProvider<FunctionCall> functionCall(
-            String name,
-            boolean distinct,
-            List<PlanTestSymbol> args)
-    {
-        return new FunctionCallProvider(false, QualifiedName.of(name), distinct, args);
-    }
-
-    public static List<Expression> toSymbolReferences(List<PlanTestSymbol> aliases, SymbolAliases symbolAliases)
+    public static List<PlanTestSymbol> toSymbolAliases(List<String> aliases)
     {
         return aliases
                 .stream()
-                .map(arg -> arg.toSymbol(symbolAliases).toSymbolReference())
+                .map(PlanMatchPattern::symbol)
                 .collect(toImmutableList());
     }
 
-    private static List<PlanTestSymbol> toSymbolAliases(List<String> aliases)
+    public static List<Expression> toSymbolReferences(List<PlanTestSymbol> aliases, SymbolAliases symbolAliases)
```

Comment:
Nit: spurious change.

Reply:
I didn't actually change anything in the signature. The changes were being shown in this way, because the order of the two methods was reversed compared to original order.